src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx

```diff
@@ -115,7 +109,7 @@ export class ConnectedFilterControl extends Component<
     const moreFiltersButtonContainerClassname = classNames(
       styles.MoreFiltersButtonContainer,
       actionsToRender.length === 0 &&
-        (newDesignLanguage || queryFieldHidden) &&
+        queryFieldHidden &&
         styles.onlyButtonVisible,
```

Comment:
Logic is failing causing regression in Percy. Might be related to button styles not being shipped 👀

Reply:
Worked it out.

When newDesignLanguage is true then `(newDesignLanguage || queryFieldHidden)` is always true, so original simplification was wrong.

Simplifying when newDesignLanguage = true:

```
actionsToRender.length === 0 && (newDesignLanguage || queryFieldHidden)
// is the same as
actionsToRender.length === 0 && (true || queryFieldHidden)
// is the same as 
actionsToRender.length === 0 && true
// is the same as 
actionsToRender.length === 0
```

NOT the original simplification `actionsToRender.length === 0 && queryFieldHidden`